main.rb

```diff
@@ -0,0 +1,9 @@
+require_relative 'app'
+
```

Comment:
- [ ] Since `main.rb` is the entry point, please adjust it to have the UI methods. You can move your `initialize`, `execute_user_option` and `run` methods from `app.rb` here. It can look similar to the code snippet below.
```
def main
  app = App.new()

  response = nil

  puts "Welcome to School Library App!\n\n"
  while response != "7"
    puts "Please choose an option by enterin a number:"
    puts "1 - List all books"
    puts "2 - List all people"
    puts "3 - Create a person"
    puts "4 - Create a book"
    puts "5 - Create a rental"
    puts "6 - List all rentals for a given person id"
    puts "7 - Exit"
    # *Lets users choose an option.
    response = gets.chomp

    case response
    when "1"
      app.list_books()
    when "2"
      app.list_people()
    when "3"
      app.create_person()
    when "4"
      app.create_book()
    when "5"
      app.create_rental()
    when "6"
      app.list_rentals_for_person_id()
    when "7"
      # *Have a way to quit the app.
      puts "Thank you for using this app!"
    end

    puts "\n"
  end
end

main()
```
***Note***: Your code does not have to follow the code snippet above. You can adjust it according to your project 👍

Reply:
Hi @Lucas-Masaba, thanks for the review. The requested changes will be implemented.